plugins/pulp_docker/plugins/importers/importer.py

```diff
@@ -220,13 +240,59 @@ def import_units(self, source_repo, dest_repo, import_conduit, config, units=Non
                 unit_filter = {'image_id': {'$in': list(units_to_add)}}
                 criteria = UnitAssociationCriteria(type_ids=[constants.IMAGE_TYPE_ID],
                                                    unit_filters=unit_filter)
-                units = import_conduit.get_source_units(criteria=criteria)
+                units = conduit.get_source_units(criteria=criteria)
             else:
                 # Break out of the loop since there were no units to add to the list
                 break
 
         return units_added
 
+    @staticmethod
+    def _import_manifests(conduit, units):
+        """
+        Import manifests and referenced blobs.
+
+        :param conduit: provides access to relevant Pulp functionality
+        :type conduit: pulp.plugins.conduits.unit_import.ImportUnitConduit
+        :param units: optional list of pre-filtered units to import
+        :type  units: list of pulp.plugins.model.Unit
+        :return: list of units that were copied to the destination repository
+        :rtype:  list
+        """
+        units_added = []
+
+        # All manifests if not specified
+
```

Comment:
My personal preference would be to eliminate this blank line so the comment is "touching" the code block it is about. No need to change.